### PR TITLE
Add ability to step in, out, and over

### PR DESCRIPTION
**Pull request by joshbtn**
I've added the ability to step in, out and over

### CODE REVIEW

**Comment by joshbtn** [comment]
debated tests on this one since the new methods don't do much, but I looking over what's already in the tests I think i'll follow suit.

File: templates/main.html [hunk 1]
```diff
@@ -11,16 +11,16 @@
         <button class='btn play' rv-on-click="togglePlay" rv-disabled="context.isDisconnected < state.current" rv-title="playText < context.state.current">
           <span rv-class="playIconClass < context.state.current"></span>
         </button>
-        <button class='btn step-over' title="Step over">
+        <button class='btn step-over' rv-on-click="stepOver" rv-disabled="context.isDisconnected < state.current" title="Step over">
```
Comment (johanlunds) [inline templates/main.html:14]: I haven't tried this PR out locally (I will though). Should this be enabled only when **paused** perhaps?

I'm thinking you might trigger a bug if you click it when the program is running. 

As I said will try out locally.

**Comment by johanlunds** [comment]
Awesome stuff, thanks!

Merging despite the comment above.

File: templates/main.html (hunk 1 above)
Comment (joshbtn) [inline templates/main.html:14]: Good point I'll test it out.

**Pull request merged**